src/Tags.zig

```diff
@@ -1,5 +1,9 @@
 const std = @import("std");
 
+const windowsApi = @import("windows.zig");
+
+const OS_TAG = @import("builtin").os.tag;
```

Comment:
Same comment here as in build.zig, use `const builtin = @import("builtin");` and use `builtin.os.tag == .windows` in the if statements.